deploy/docker/docker-entrypoint.sh

```diff
@@ -126,7 +146,7 @@ do
         # Config in plugins/*
         for CONFIG_PLUGINS_FILE in $(ls $CONFIG_PLUGINS); do
             if [[ ! -z "$(cat $CONFIG_PLUGINS/$CONFIG_PLUGINS_FILE |grep -E "^(^|^#*|^#*\s*)$VAR_NAME")" ]]; then
-                echo "$VAR_NAME=$(eval echo \$$VAR_FULL_NAME)"
+                echo "$VAR_NAME=$(echo_value)"
```

Comment:
`echo_value`

Reply:
resolved in 52c02a71dfc7f3331a1a80f5fcb52b9c42f410cc